engine/src/test/java/org/camunda/bpm/engine/test/api/mgmt/BatchStatisticsQueryTest.java

```diff
@@ -257,6 +261,38 @@ public void testQueryOrderByIdDec() {
     verifySorting(statistics, inverted(batchStatisticsById()));
   }
 
+  @Test
+  public void testQueryOrderByStartTimeAsc() {
+    // given
+    helper.migrateProcessInstancesAsync(1);
+    ClockUtil.setCurrentTime(new Date(ClockUtil.getCurrentTime().getTime() + 7200000L));
```

Comment:
The date calculation is hard to understand, it's tough for me to understand the value `7200000L`. A helper method with a speaking name (or abstracting millisecond conversion) would be clearer.

Reply:
I made it easier to understand.